http/src/response/future.rs

```diff
@@ -176,9 +176,9 @@ impl InFlight {
 struct RatelimitQueue {
     guild_id: Option<Id<GuildMarker>>,
     invalid_token: Option<Arc<AtomicBool>>,
+    response: HyperResponseFuture,
+    timeout: Duration,
```

Comment:
I feel the previous names were quite self-descriptive. `response` is not a response, it's a future resolving to a response. I think this is an important distinction to be making in a wrapper around a future resolving to a response.